charts/db-operator/templates/controller/deployment.yaml

```diff
@@ -4,16 +4,26 @@ kind: Deployment
 metadata:
   name: {{ template "db-operator.name" . }}
   labels:
-    {{- include "db-operator.labels" . | nindent 4 }}
+    app: {{ template "db-operator.name" . }}
```

Comment:
If we go this kind of labels, I don't see why having both `app` and `app.k8s.io./name`, maybe only second is enough.

Reply:
I deleted the `app` labels in https://github.com/db-operator/charts/pull/13/commits/0302ef6e403ca1ac01fa7a575d839c124ef4793b